examples/webpack.config.local.js

```diff
@@ -11,48 +11,61 @@ const ALIASES = require('ocular-dev-tools/config/ocular.config')({
   root: resolve(__dirname, '..')
 }).aliases;
 
+const ROOT_DIR = resolve(__dirname, '..');
+
+const MATH_LINK_ALIASES = {
+  'math.gl': resolve(ROOT_DIR, '../math.gl/modules/core/src'),
+  '@math.gl/culling': resolve(ROOT_DIR, '../math.gl/modules/culling/src'),
+  '@math.gl/geospatial': resolve(ROOT_DIR, '../math.gl/modules/geospatial/src')
+};
+
 // eslint-disable-next-line import/no-extraneous-dependencies
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
 // Support for hot reloading changes to the library:
-const LOCAL_DEVELOPMENT_CONFIG = {
-  mode: 'development',
+function makeLocalDevConfig(linkToMath) {
+  const alias = linkToMath ? Object.assign({}, ALIASES, MATH_LINK_ALIASES) : ALIASES;
+  return {
+    mode: 'development',
+
+    devtool: 'source-map',
+
+    // suppress warnings about bundle size
+    devServer: {
+      stats: {
+        warnings: false
+      },
+      contentBase: [resolve('.'), resolve('../../')]
+    },
 
-  devtool: 'source-map',
+    // this is required by draco
+    node: {
+      fs: 'empty'
+    },
 
-  // suppress warnings about bundle size
-  devServer: {
-    stats: {
-      warnings: false
+    resolve: {
+      // Imports the library from its src directory in this repo
+      alias
     },
-    contentBase: [resolve('.'), resolve('../../')]
-  },
-
-  // this is required by draco
-  node: {
-    fs: 'empty'
-  },
-
-  resolve: {
-    // Imports the library from its src directory in this repo
-    alias: ALIASES
-  },
-
-  module: {
-    rules: [
-      {
-        // Unfortunately, webpack doesn't import library sourcemaps on its own...
-        test: /\.js$/,
-        use: ['source-map-loader'],
-        enforce: 'pre'
-      }
-    ]
-  },
-
-  plugins: [new webpack.EnvironmentPlugin(['MapboxAccessToken'])]
-};
 
-function addLocalDevSettings(config) {
+    module: {
```

Comment:
I think it is good that this is controlled by a webpack env command line param.

But maybe just add a second function (`addLocalLibSettings`) to avoid making the already big `addLocalDevSetting` function more complicated.

That would minimize the delta between this webpack.local file and similar webpack.local config files in other repos.